its/src/test/java/com/sonar/it/scanner/msbuild/ScannerMSBuildTest.java

```diff
@@ -937,28 +951,73 @@ public void incrementalPrAnalysis_NoCache() throws IOException {
 
   @Test
   public void incrementalPrAnalysis_ProducesUnchangedFiles() throws IOException {
-    // ToDo: Compute hashes of files and store them to protobuf data
-    // ToDo: Populate server cache for "base-branch". File3 should get wrong hash. Might need change of license edition on Orchestrator
     String projectKey = "incremental-pr-analysis";
+    String baseBranch = "master";
```

Comment:
When was the cache API introduced? Was it available on 7.9 and 8.9? We might need to bail out for old versions.
See `testNoProjectNameAndVersion` and this to bail out as inconclusive test
```
Assume.assumeTrue(ORCHESTRATOR.getServer().version().isGreaterThanOrEquals(6, 1));
```

Reply:
Good point. It was implemented in 9.4: https://sonarsource.atlassian.net/browse/SONAR-16097